### PR TITLE
Added utils for testing release packages

### PR DESCRIPTION
**Pull request by btoll**
check_sig: Verify gpg signatures of build artifacts.

test_package: Verifies the packages were built from the correct branch
with the correct hash and verifies the test version release number.

### CODE REVIEW

File: scripts/release/util/check_sig.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+# shellcheck disable=2045
+
+if [ $# -eq 0 ]
```
Comment (tsachiherman) [inline scripts/release/util/check_sig.sh]: you want to check that you have a single argument, rather then checking that the user forgot to specify a directory.

File: scripts/release/util/smoke_test.sh [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            shift
+            RELEASE="$1"
+            ;;
+        *)
+            echo "Unknown option" "$1"
+            exit 1
+            ;;
+    esac
+    shift
+done
```
Comment (tsachiherman) [inline scripts/release/util/smoke_test.sh]: could you please add verification that all the above parameters have been provided ( either via export or explicitly provided as command line arguments ? )

File: scripts/release/util/test_package.sh [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+            shift
+            RELEASE="$1"
+            ;;
+        *)
+            echo "$RED_FG[$0]$END_FG_COLOR Unknown option $1"
+            exit 1
+            ;;
+    esac
+    shift
+done
```
Comment (tsachiherman) [inline scripts/release/util/test_package.sh]: same as before; make sure that these arguments are set before proceeding further.

File: scripts/release/util/check_sig.sh (hunk 1 above)
Comment (btoll) [reply scripts/release/util/check_sig.sh]: Will change.

File: scripts/release/util/smoke_test.sh (hunk 2 above)
Comment (btoll) [reply scripts/release/util/smoke_test.sh]: Will do.

File: scripts/release/util/test_package.sh (hunk 3 above)
Comment (btoll) [reply scripts/release/util/test_package.sh]: Will do.

**Pull request merged**